test/functional/test-fixed-layer.js

```diff
@@ -376,6 +376,28 @@ describe('FixedLayer', () => {
       expect(fe.element.style.top).to.equal('calc(17px + 11px)');
     });
 
+    it('should reset top upon being removed from fixedlayer', () => {
+      expect(fixedLayer.fixedElements_).to.have.length(2);
+
+      // Add.
+      fixedLayer.addElement(element3, '*');
+      expect(fixedLayer.fixedElements_).to.have.length(3);
+      const fe = fixedLayer.fixedElements_[2];
+      expect(fe.id).to.equal('F2');
+      expect(fe.element).to.equal(element3);
+      expect(fe.selectors).to.deep.equal(['*']);
+
+      // Remove.
+      fixedLayer.vsync_ = {
+        mutate: function(callback) {
+          callback();
+        },
+      };
+      fixedLayer.removeElement(element3);
+      expect(fixedLayer.fixedElements_).to.have.length(2);
+      expect(element3.style.top).to.equal('');
```

Comment:
We should check if the `top != ''` first :) Or, alternatively, we can fill it in.